app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
```

Comment:
This is just raw devise generate form styles, you need to add you CSS classes to make the login look as a part of your app :)